.github/workflows/windows.yml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2020, 219 Design, LLC
+# See LICENSE.txt
+#
+# https://www.219design.com
+# Software | Electrical | Mechanical | Product Design
+#
+name: Win_CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: windows-2019
```

Comment:
I claim that this file (windows.yml) is "done".  I don't think any further changes are needed here.